travis/.travis.yml

```diff
@@ -29,6 +29,7 @@ rvm:
   - jruby-9.1.7.0 # pin JRuby to this until travis/rvm can install later versions
   - jruby-head
   - jruby-1.7
+  - truffleruby
```

Comment:
This is going to be interesting with `rspec-rails`, we're hitting the limit of concurrent builds on Circle in it, and struggling to keep in certain limits not to wait for the result for half an hour.

What are your plans for other RSpec repos?

Reply:
Actually we'd like to add TruffleRuby to individual gems as they pass, and initially focus on rspec-core, expectations, mocks, and support.
When we get to `rspec-rails` I think we can reconsider that. One possibility might be to run it daily instead of on push, or to use another CI service for running TruffleRuby (e.g. GitHub Actions).